app/components-react/sidebar/NavTools.tsx

```diff
@@ -91,64 +94,159 @@ export default function SideNav() {
-      {isLoggedIn && !isPrime && (
-        <div
-          className={cx(styles.cell, styles.primeCell)}
-          onClick={upgradeToPrime}
-          title={$t('Get Prime')}
-        >
-          <Badge count={<i className={cx('icon-pop-out-3', styles.linkBadge)} />}>
-            <i className="icon-prime" />
-          </Badge>
-        </div>
-      )}
-      {isLoggedIn && (
-        <div
-          className={styles.cell}
-          onClick={() => throttledOpenDashboard()}
-          title={$t('Dashboard')}
-        >
-          <Badge count={<i className={cx('icon-pop-out-3', styles.linkBadge)} />}>
-            <i className="icon-dashboard" />
-          </Badge>
-        </div>
-      )}
-      {isLoggedIn && (
-        <div
-          className={cx(styles.cell)}
-          onClick={() => throttledOpenDashboard('cloudbot')}
-          title={$t('Cloudbot')}
-        >
-          <Badge count={<i className={cx('icon-pop-out-3', styles.linkBadge)} />}>
-            <i className="icon-cloudbot" />
-          </Badge>
-        </div>
-      )}
-      <div className={styles.cell} onClick={openLayoutEditor} title={$t('Layout Editor')}>
-        <i className="fas fa-th-large" />
-      </div>
-      <div
-        className={cx(styles.cell, { [styles.toggleOn]: studioMode })}
-        onClick={toggleStudioMode}
-        title={$t('Studio Mode')}
+    <>
+      <Menu
+        key={ENavName.BottomNav}
+        forceSubMenuRender
+        mode="inline"
+        className={cx(styles.bottomNav, !isOpen && styles.closed, isOpen && styles.open)}
+        defaultOpenKeys={openMenuItems && openMenuItems}
       >
-        <i className="icon-studio-mode-3" />
-      </div>
-      <div className={styles.cell} onClick={openHelp} title={$t('Get Help')}>
-        <Badge count={<i className={cx('icon-pop-out-3', styles.linkBadge)} />}>
-          <i className="icon-question" />
-        </Badge>
-      </div>
-      <div className={styles.cell} onClick={openSettingsWindow} title={$t('Settings')}>
-        <i className="icon-settings" />
-      </div>
-    </div>
+        {menuItems.map((menuItem: IParentMenuItem) => {
+          if (isDevMode && menuItem.title === EMenuItem.DevTools) {
+            return (
+              <Menu.Item
+                key={menuItem.key}
+                title={menuItem.title}
+                icon={<i className="icon-developer" />}
+                onClick={openDevTools}
+              >
+                {menuItem.title}
+              </Menu.Item>
+            );
+          } else if (isLoggedIn && !isPrime && menuItem.title === EMenuItem.GetPrime) {
+            return (
+              <Menu.Item
+                key={menuItem.key}
+                title={$t(menuItem.title)}
+                icon={
+                  <div>
+                    <Badge count={<i className={cx('icon-pop-out-3', styles.linkBadge)} />}>
+                      <i className={menuItem.icon} />
+                    </Badge>
+                  </div>
```

Comment:
unnecessary div?

Reply:
Without this div, the icon property on an antd menu item won't apply the transformation to translate the popout icon over the menu item icon in the collapsed menu. In fact, if won't apply an initial transformation of any kind on the menu item icon in the collapsed menu without this wrapping div (I experimented with scaling).
![badge-div](https://user-images.githubusercontent.com/69046953/197850448-481ba3f2-2917-444e-b8c3-ce7087540ce2.png)